assets/components/prometheus-operator/manifests/templates/grafana/dashboards-1.14/k8s-coredns.yaml

```diff
@@ -80,15 +80,15 @@ data:
                 "steppedLine": false,
                 "targets": [
                     {
-                        "expr": "sum(rate(coredns_dns_request_count_total{instance=~\"$instance\"}[5m])) by (proto)",
+                        "expr": "sum(rate(coredns_dns_requests_total{instance=~\"$instance\"}[5m])) by (proto)",
```

Comment:
Is this patch needed? We shouldn't be using `-1.14` dashboards I think.

Reply:
The way I understand it is all the dashboards in that dir are for k8s that are above 1.14, see the checks in those files:

```yaml
{{- if and (semverCompare ">=1.14.0-0" $kubeTargetVersion) (semverCompare "<9.9.9-9" $kubeTargetVersion) 
```

Reply:
This patch is needed because the metrics that are on the dashboard are not what coredns releases. Also we have issue with coredns: https://github.com/kinvolk/lokomotive/issues/374

Reply:
Ah yes, it seems to be the other way around. Other dashboards are >=1.10 <=1.14 :facepalm: what a poor naming. All good, thanks for checking!